src/widget/taskBar.js

```diff
@@ -11,7 +11,9 @@ const { ShellVersionMatch } = Me.imports.src.utils.compatibility;
 const { MsWindow } = Me.imports.src.layout.msWorkspace.msWindow;
 const { reparentActor } = Me.imports.src.utils.index;
 
-let dragData = null;
+var dragData = {
+    current: null
```

Comment:
Could you enlight me about the need to regroup everything under the new `current` object?

Reply:
Its data was not exported to workspaceList.js otherwise. Perhaps simple variables can't be exported in GJS?

Reply:
Ok but why do we need to export this variable since the only time we use it it's to retrieve the 'item' which is the same as the 'source' parameters no?

Reply:
Oh, got it. The other code removed some of the source paramentes, I didn't realize it worked that way. I will change the code. Thanks.